### DISCUSSION TITLE
Force fields for materials/battery applications

### DISCUSSION THREAD

**Discussion by davidlmobley** [openforcefield/discussions#19] [Ideas]
We received something like this via e-mail:

>  I am ... conducting research work related to batteries. I am writing to request your assistance in obtaining a force field file of combine Cr, Li and Te.
>
>I have been utilizing various resources in my research, but unfortunately, I have reached a point where I cannot proceed without this specific file. Therefore, I am kindly requesting that you provide me with the combine force field file of Cr , Li and Te to help me continue with my work.

Short answer: No, we cannot do this at present.

Long version: While this is very interesting, this is not currently within scope for OpenFF, which is mostly focused on biomolecular force fields, at least with its current funding.

Some of our infrastructure can be extended to cover more general chemistries and additional functional forms; for example, Brian Space's group at North Carolina State is working on advanced functional forms using OpenFF infrastructure, including for more materials-oriented applications. That said, I think there are still many research questions to be answered before general force fields can be used in this space. 

Feel free to extend this thread with other discussion relevant to this area.

**Comment by mrshirts** [comment]
Transition metals are hard and likely we will be unable to do it within the existing framework.  If one is looking at different oxidation states of Li, that also could be quite hard.
